packages/ui/__stories__/SimpleTable.module.scss

```diff
@@ -0,0 +1,13 @@
+.th {
+  background: #68c200;
```

Comment:
hardcoded colors. not available as constants? 🤔

Reply:
This is custom story style. I will rename the file